### PR TITLE
Backend/initial configurations

### PR DESCRIPTION
**Pull request by Sorin37**

### CODE REVIEW

File: Backend/Backend/Interfaces/IMongoDBSettings.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Backend.Interfaces
+{
+    public interface IMongoDBSettings
+    {
+        string SlotsCollectionName { get; set; }
+        string ConnectionString { get; set; }
+        string DatabaseName { get; set; }
+
```
Comment (TinelValentin) [inline Backend/Backend/Interfaces/IMongoDBSettings.cs:8]: Space

File: Backend/Backend/Controllers/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            if (users == null)
+            {
+                return NotFound();
+            }
+            else
+            {
+                return Ok(users);
+            }
+        }
+
```
Comment (BrassoiCatalin) [inline Backend/Backend/Controllers/UserController.cs:31]: Too much blank; one row is enough.

File: Backend/Backend/Program.cs [hunk 3]
```diff
@@ -1,8 +1,19 @@
+using Backend.Interfaces;
+using Backend.Models;
+using Backend.Services;
+using Backend.Settings;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
```
Comment (TinelValentin) [inline Backend/Backend/Program.cs]: Either delete this new line or put one after each comment

File: Backend/Backend/Services/UserService.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
+﻿using Backend.Interfaces;
+using Backend.Models;
+using MongoDB.Driver;
+
+namespace Backend.Services
+{
+
```
Comment (TinelValentin) [inline Backend/Backend/Services/UserService.cs:7]: Line

File: Backend/Backend/Controllers/UserController.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    public class UserController : Controller
+    {
+        IUserService _userService;
+
+        public UserController(IUserService userService)
+        {
+            _userService = userService ?? throw new ArgumentNullException(nameof(userService));
+        }
+
+
```
Comment (TinelValentin) [inline Backend/Backend/Controllers/UserController.cs]: Line

File: Backend/Backend/Controllers/UserController.cs [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            {
+                return NotFound();
+            }
+            else
+            {
+                return Ok(users);
+            }
+        }
+
+
```
Comment (TinelValentin) [inline Backend/Backend/Controllers/UserController.cs]: Line

**Pull request merged**